library/src/main/res/layout/chucker_transaction_item_copy.xml

```diff
@@ -0,0 +1,17 @@
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:gravity="end">
+
+    <ImageButton
```

Comment:
Can we tint this as the rest of the text?
As it looks weird that this icon is dark black while the rest of the text is not

Reply:
changes done 
![Screenshot_1732956576](https://github.com/user-attachments/assets/24d6df2b-8c05-4ef5-9182-5430aba73ddf)